module/os/linux/zfs/zpl_ctldir.c

```diff
@@ -55,7 +55,7 @@ zpl_root_iterate(struct file *filp, zpl_dir_context_t *ctx)
 	zfsvfs_t *zfsvfs = ITOZSB(file_inode(filp));
 	int error = 0;
 
```

Comment:
Should the zpl_ files perhaps #undef ZFS_ENTER/ZFS_EXIT so this mistake can't be reintroduced in the future?

Reply:
I suppose we could, though I'm not sure that's really needed.  Parts of this were accidentally introduced by the major restructuring of the code base which was a lot of churn and isn't a frequent activity.  I think we can avoid reintroducing this without the undefs.